Zend/Optimizer/dfa_pass.c

```diff
@@ -325,6 +325,11 @@ static bool opline_supports_assign_contraction(
 		return 0;
 	}
 
+	if (opline->opcode >= ZEND_FRAMELESS_ICALL_1 && opline->opcode <= ZEND_FRAMELESS_ICALL_3) {
+		/* Frameless calls override the return value, but the return value may overlap with the arguments. */
+		return opline->op1_type != IS_CV || opline->op1.var != cv_var;
```

Comment:
Can the exact same issue not arise with op2? I.e. 

```
CV0($value) = FRAMELESS_ICALL_2(min) int(100) CV0($value)
```

Clearing result would clear op2.

Reply:
Yes indeed, I don't know why I missed this sorry